### ISSUE TITLE
How to play multiple audios at the same time?

### ISSUE THREAD

**Issue by superbluecool** [martijn00/XamarinMediaManager#336]
See title, how to play multiple audios at the same time? I just need to play audios, and no need for any videos.

**Comment by JoshuaNovak919** [comment]
I also need the ability to play multiple audio files at once (a song, and then sound effects). Anyone know if there is a way to do this yet or if this is something coming eventually?

**Comment by mike-rowley** [comment]
I don't believe this is possible for MediaPlayer on android, it might be possible with ExoPlayer V2 but I am not sure if we will support that in the XMM V1 release.
